controllers/phjob_scheduler.go

```diff
@@ -157,12 +165,12 @@ func (r *PHJobScheduler) getGroupRemainingQuota(phJob *primehubv1alpha1.PhJob) (
 	return groupRemainingQuota, nil
 }
 
-func (r *PHJobScheduler) getUserRemainingQuota(phJob *primehubv1alpha1.PhJob) (*ResourceQuota, error) {
+func (r *PHJobScheduler) getUserRemainingQuotaInGroup(phJob *primehubv1alpha1.PhJob) (*ResourceQuota, error) {
 	groupInfo, err := r.getGroupInfo(phJob.Spec.GroupId)
 	if err != nil {
 		return nil, err
 	}
-	userUsage, err := r.getCurrentUsage(phJob.Namespace, user_aggregation_key, phJob.Spec.UserName)
+	userUsage, err := r.getCurrentUsage(phJob.Namespace, []string{group_aggregation_key, user_aggregation_key}, []string{phJob.Spec.GroupName, phJob.Spec.UserName})
```

Comment:
group_aggregation_key and user_aggregation_key should be CamelCase

Reply:
Fixed. Thanks!